### ISSUE TITLE
.product editor outline does not react to sort button

### ISSUE THREAD

**Issue by Bananeweizen** [eclipse-pde/eclipse.pde#172]
The outline of the product file editor does not correctly sort items when the sort button is pressed.
<img width="352" alt="image" src="https://user-images.githubusercontent.com/406876/174780875-0300cac8-dad2-4efe-be76-f94fef8cbaae.png">

**Comment by Bananeweizen** [comment]
No. That specific outline page actively ignores sorting by implementing a no-sort.

**Comment by HannesWell** [comment]
Link to the PR did not work...

Fixed via https://github.com/eclipse-pde/eclipse.pde/pull/173.

**Comment by Bananeweizen** [comment]
> Link to the PR did not work...

Thanks for mentioning that. I've done this multiple times now, and I have the feeling that it doesn't work at all when put inside braces. I'll try to remember using something like "actual message - fixes number" next time.

**Comment by HannesWell** [comment]
> > Link to the PR did not work...
> 
> Thanks for mentioning that. I've done this multiple times now, and I have the feeling that it doesn't work at all when put inside braces. I'll try to remember using something like "actual message - fixes number" next time.

Putting  #<issue-number> in braces does not work from my experience as well. But what worked from my experiance is to write "Fixes" or "Resolves" followed by the full issue URL into the commit message. This also has the nice side-effect that you can find the issue more easily since you can just click on the link.

Nevertheless according to the doc (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) my comment should have linked the PR. But I noticed I can also do that manually in the sidebar (what I have just done).
